ingestion-beam/pom.xml

```diff
@@ -92,6 +74,14 @@
             <groupId>org.apache.beam</groupId>
             <artifactId>beam-sdks-java-extensions-json-jackson</artifactId>
         </dependency>
+        <dependency>
+            <!-- Must be defined before beam-sdks-java-io-google-cloud-platform because they depend
+                 on different versions of google-cloud-pubsublite to implement PubsubLiteIO, and
+                 this version supports messages larger than 1MB -->
+            <groupId>com.google.cloud</groupId>
+            <artifactId>pubsublite-beam-io</artifactId>
+            <version>0.13.1</version>
+        </dependency>
```

Comment:
I assume that beam-sdks-java-io-google-cloud-platform will eventually update to a pubsublite version such that explicitly defining this dependency will no longer be necessary. Is that correct?

Reply:
yes